api-specificatie/openapi.yaml

```diff
@@ -688,9 +619,11 @@ paths:
         Parameter adresseerbaarobjectidentificatie-query bevat de 16 cijferige identificatie van een ligplaats, standplaats of verblijfsobject"
       parameters:
         - $ref: '#/components/parameters/adresseerbaarobjectidentificatie-query'
+        - $ref: '#/components/parameters/locatie-query'
```

Comment:
er zit een koppelteken in de componentnaam (komt nu ook al voor). @MelvLee heb je daar bij genereren geen last van?

Reply:
Dit is een $ref. In de geresolvede versie komt het niet voor